frontend/services/popupNotifications.service.js

```diff
@@ -0,0 +1,74 @@
+(function () {
+  "use strict";
+
+  angular
+    .module("base")
+    .service("popupNotifications", popupNotifications);
+
+  popupNotifications.$inject = ["Alertify"];
+
+  function popupNotifications(Alertify) {
+    var vm = this;
+    vm.notifyAlert = notifyAlert;
+    vm.notifyConfirm = notifyConfirm;
+    vm.notifySuccess = notifySuccess;
+    vm.notifyStandart = notifyStandart;
+    vm.notifyError = notifyError;
+    vm.notifyPersistent = notifyPersistent;
+    vm.notifyNewMsg = notifyNewMsg;
+
+    function notifyAlert(alertMsg) {
+      alertMsg = alertMsg || "Hello in Farm Apps";
+      Alertify.alert(alertMsg);
+    }
+
+    function notifyConfirm(confirmMsg) {
```

Comment:
here it is possible to modify button text, so implement this optional functionality